DataProcessing/data_tagger.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pickle
+from collections import defaultdict
+from matplotlib import pyplot as plt
+
+
+# todo all of these actions need to be done to the db, currently they all run in O(n) for each Crop changed !
+from DataProcessing.dataProcessingConstants import ID_TO_NAME
+
+
+def mark_vague(db, track, crop_inds):
+    for crop_id in crop_inds:
+        db_index = db.index(track[crop_id])
+        db[db_index].is_vague = True
+
+def relabel_all(db, track, new_label):
+    for crop in track:
+        db_index = db.index(crop)
+        db[db_index].set_label(new_label)
+
+def discard_crops(db, track, crop_inds):
+    # todo this should be done later to keep presistance of ids, or at the end of the track
+    for crop_id in crop_inds:
+        db_index = db.index(track[crop_id])
+        del db[db_index] # todo should we del the actual crops as well?
+
+def split_track(db, track, split_start, split_end , new_label):
+    splitted_track = track[split_start:split_end]
+    new_track_id = max(crop.track_id for crop in db) + 1
```

Comment:
Needs to be a global variable so we need to look for the maximum only once at the beginning of the current video.